packages/beacon-node/src/node/nodejs.ts

```diff
@@ -74,6 +75,12 @@ enum LoggerModule {
   sync = "sync",
 }
 
+/**
+ * Short delay before closing db to give async operations sufficient time to complete
+ * and prevent "Database is not open" errors when shutting down beacon node.
```

Comment:
As per [documentation](https://github.com/Level/classic-level#dbclosecallback), "Database is not open" only happens if close is already called and some async operation still tries to execute db queries. If db query is already executed before, the database will wait for any pending operations to finish before closing.